vnext/src/IntegrationTests/WebSocketBinaryTest.js

```diff
@@ -61,6 +61,10 @@ class WebSocketBinaryTest extends React.Component<{...}, State> {
   };
 
   _connect = () => {
+    // Test protocols parsing. No further use.
+    const scalarProtocolSocket = new WebSocket(this.state.url, "p0");
```

Comment:
Are these used anywhere?

Reply:
Oh, I see the comment now.

Note that creating the object here won't synchronously call the code in native. So if this triggers an action, we may see random crashes later in the test. Would be great to find some signal to wait on here.

Reply:
Maybe asserting a correctly constructed state in the end of the test should be enough. I'll give it a try.